### DISCUSSION TITLE
Office IP blocked for some project

### DISCUSSION THREAD

**Discussion by madsbuch** [vercel/community#3019] [Help] [answered]
### Summary

We have a number of projects that run on vercel. One of the projects is a rewriting service for our backend. The backend runs on port 445, but not all clients like non-standard ports, so we rewrite mobile.api.domain:443 to api.domain:445.

I did some load testing of our API yesterday. Don't know if this has anything to dow with it.

Anyways, I can't connect to mobile.api.domain anymore. It simply closes the connection.

### Example

None

### Steps to Reproduce

None

**Comment by swarnava** [comment, marked as answer]
Hi,

Load testing is prohibited for Non Enterprise users and it require prior approval: https://vercel.com/guides/what-s-vercel-s-policy-regarding-load-testing-deployments.

Your IP will be automatically unblocked after 72 hours. You can raise a request incase it doesn't get unblocked after 72 hours: https://vercel.com/help

**Reply by madsbuch** [reply]
Thanks a lot! I was not aware of this. We will wait it out :)
